src/pages/verify/integrations/react-native-expo.mdx

```diff
@@ -11,41 +11,44 @@ export default Layout;
 
 Read the [full documentation](https://github.com/criipto/criipto-verify-expo) or [download a sample application from GitHub](https://github.com/criipto/criipto-verify-expo/tree/master/example).
 
-## App switch support
+## Register Your Application in Criipto Verify
 
-`@criipto/verify-expo` supports app switching for Swedish BankID and Danish MitID.
+After you signed up for Criipto Verify, you must [register an application](https://dashboard.criipto.com/applications/add) before you can try logging in with any eID.
 
-### Danish MitID + Android
+Once your application is registered, the dashboard will contain the necessary details to configure your application to communicate with Criipto Verify.
+You will need the following information from the General tab of your application settings:
 
-Switchback from the Danish MitID mobile application will only work if you are using a universal link / app link as your redirect_uri.
+- _Client ID_ to identify your application to Criipto Verify.
+- _Domain_ on which you will be communicating with Criipto Verify.
 
-[Guide to Expo universal links](https://docs.expo.dev/guides/deep-linking/).
+### Register callback URLs
 
-Enable switchback on Android by adding the following to your `app.json`:
+Before you can start sending authentication requests to Criipto Verify, you need to register the URLs on which you want to receive the returned _JSON Web Token_, [JWT](verify/getting-started/token-contents/).
 
-```json
-"plugins": [
-  ["@criipto/verify-expo", {
-    "androidAppLinks": [
-      "https://..."
-    ]
-  }]
-]
-```
+The callback URL of your application is the URL where Criipto Verify will redirect to after the user has authenticated in order for the OpenID Connect middleware to complete the authentication process.
 
-#### Expo Go
+You will need to add this URL to the list of allowed URLs for your application.
 
-Danish MitID on Android will not work with Expo Go due to the use of `createTask: false`. You must use a build to test, for instance with `npx expo run:android`
+For example, if you are running your application locally, the callback URL could be `http://localhost:19006/`. Alternatively, if developing in Expo Go, the callback URL may look like this: `exp://127.0.0.1:8081`.
```

Comment:
It will never be a `http://` url when developing a mobile application (and we don't support Expo web)

Reply:
Might want to provide examples using custom schemes instead,